### PR TITLE
smtpmail: Allow to not reset all configs and packets at the end of the script

### PR DESCRIPTION
**Pull request by szaimen**
As the title says

### CODE REVIEW

File: addons/smtp-mail.sh [hunk 1]
```diff
@@ -218,7 +218,7 @@ ALIASES_CONF
 
 # Define the mail-program
 cat << DEFINE_MAIL > /etc/mail.rc
-set sendmail="/usr/bin/msmtp"
```
Comment (szaimen) [inline addons/smtp-mail.sh:221]: This doesn't work for me so I restored the old config.

Comment (szaimen) [reply addons/smtp-mail.sh:221]: I have an idea how to test both.

Comment (szaimen) [reply addons/smtp-mail.sh:221]: done

**Comment by szaimen** [comment]
Now i've made it possible to test both variants before failing. 
It was already successfully tested by myself.
I hope this is okay now :)

**Comment by enoch85** [comment]
OK, much better!

**Pull request merged**